src/js/util/sort_data.js

```diff
@@ -0,0 +1,34 @@
+import moment from 'moment';
+
+export function sortData1(apiData) {
```

Comment:
Could you please rename these functions so it's clearer what they actually do? Sort data how...?